docs/oadp-backups.md

```diff
@@ -0,0 +1,225 @@
+```
+oc create sa job-runner
+```
+
+- Grant OADP NS Permissions
+The job-runner SA will need to be able to create and monitor resources in the openshift-adp namespace so we give it permission to do so.
+```
+oc project openshift-adp
+oc adm policy add-role-to-user admin -z job-runner
+```
+
+- Grant MTA NS Permissions
+The job-runner SA will also need to shutdown, start, and monitor resources in the openshift-mta namespace so, once again give it permission to do so
+```
+oc project openshift-mta
+oc adm policy add-role-to-user admin system:serviceaccount:openshift-adp:job-runner
+```
+
+- Create a CronJob
+Note that the time is cluster time, which by default is UTC.
+```
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: mta-backup
+  namespace: openshift-adp 
+spec:
+  schedule: "03 16 * * 1-5"
+  concurrencyPolicy: Allow
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: alpine-container
+            image: registry.redhat.io/openshift4/ose-cli:latest
+            command:
+              - bash
+              - -c
+              - |
+                cd /tmp
+                curl -L -O https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64
+                mv jq-linux-amd64 jq
+                chmod +x jq
+                oc scale -n openshift-mta deployment rhsso-operator --replicas=0
+                oc wait -n openshift-mta po -l name=rhsso-operator --for=delete
+                oc scale -n openshift-mta deployment mta-operator --replicas=0
+                oc wait -n openshift-mta po -l name=mta-operator --for=delete
+                oc scale -n openshift-mta statefulset keycloak --replicas=0
+                oc wait -n openshift-mta po -l app=keycloak --for=delete
+                oc scale -n openshift-mta deployment mta-keycloak-postgresql-15 --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-keycloak-postgresql-15 --for=delete
+                oc scale -n openshift-mta deployment mta-ui --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-ui --for=delete
+                oc scale -n openshift-mta deployment mta-hub --replicas=0
+                oc wait -n openshift-mta po -l app.kubernetes.io/name=mta-hub --for=delete
+                export BACKUP_DATE=$(date +%Y-%m-%d-%H-%M-%S)
+                cat << EOF | oc apply -f -
+                kind: Backup
+                apiVersion: velero.io/v1
+                metadata:
+                  name: backup-$BACKUP_DATE
+                  namespace: openshift-adp
+                spec:
+                  csiSnapshotTimeout: 30m0s
+                  defaultVolumesToFsBackup: false
+                  includedNamespaces:
+                  - openshift-mta
+                  itemOperationTimeout: 30h0m0s
+                  snapshotMoveData: true
+                  backupLocation: velero-1
```

Comment:
Remove `backupLocation` (not a spec in backup API)
you already have `storageLocation` specified.